<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" tests="9" skipped="0" failures="0" errors="0" timestamp="2023-03-20T07:59:10" hostname="DESKTOP-VGJP1PJ" time="0.015">
  <properties/>
  <testcase name="resolveMethodFromAnnotation()" classname="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" time="0.006"/>
  <testcase name="resolveMethodInherited()" classname="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" time="0.001"/>
  <testcase name="resolveMethodFromArgument()" classname="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" time="0.0"/>
  <testcase name="ambiguousExceptionMapping()" classname="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" time="0.001"/>
  <testcase name="resolveMethodExceptionSubType()" classname="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" time="0.001"/>
  <testcase name="resolveMethodBestMatch()" classname="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" time="0.0"/>
  <testcase name="noExceptionMapping()" classname="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" time="0.001"/>
  <testcase name="resolveMethodNoMatch()" classname="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" time="0.0"/>
  <testcase name="resolveMethodExceptionCause()" classname="org.springframework.web.method.annotation.ExceptionHandlerMethodResolverTests" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
